guides/source/action_controller_overview.md

```diff
@@ -585,8 +585,6 @@ class CommentsController < ApplicationController
 end
 ```
 
-Note that while for session values you set the key to `nil`, to delete a cookie value you should use `cookies.delete(:key)`.
```

Comment:
We still need to retain the differentiation. Can change it to

> Note that while for session values you can set the key to `nil` to delete it, to delete a cookie value you should use `cookies.delete(:key)`.

Reply:
Changed to your suggestion.